blueprints/emberfire/files/config/environment.js

```diff
@@ -0,0 +1,48 @@
+/* jshint node: true */
```

Comment:
I couldn't find a good way to edit an existing `config/environment` so this overwrites an existing one to include the firebaseUrl. If anyone can tell me a better way to do this I'd be super grateful @tstirrat @stefanpenner